spec/features/item_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'item'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.